ipyleaflet/here.py

```diff
@@ -84,11 +86,12 @@ def build_tiles_url(**kwargs):
         return tiles_url
 
     raise ValueError(
-        "HERE APIKEY or APP_CODE/APP_ID needed to be in params or env. vars.")
+        "HERE APIKEY or APP_CODE/APP_ID needed in params or env. variables. "
+        "For more information please type 'help(build_tiles_url'.")
```

Comment:
There seem to be a missing closing parenthesis in the message: `help(build_tiles_url)`

Reply:
Well spotted. Fixed.